cypress/api/notifications.api.ts

```diff
@@ -0,0 +1,9 @@
+import { E2E_EMAIL_SITE_API_URL } from "./constants"
+
+const BASE_URL = `${E2E_EMAIL_SITE_API_URL}/notifications`
+/**
+ * NOTE: This uses the **cookie's** site and email to determine which notifications to mark as read.
```

Comment:
hey am confused about this comment here :( 
`cy.request("POST", BASE_URL)` works as a network inceptor at the network layer, how are we using a cookies's info for this ah

Reply:
`cy.request` **does not work as an interceptor** - gonna copy paste the [docs](https://docs.cypress.io/api/commands/request#Cookies) here for clarity and if there's still unclear stuff, i'm more than happy to help!

tl;dr: `request` makes it perform similarly (dk if it's identically) to a user request from a browser

![Screenshot 2023-06-14 at 12 02 20 PM](https://github.com/isomerpages/isomercms-frontend/assets/44049504/6d668379-0f75-4bff-9eb0-125d46a85d87)

Reply:
got it, thanks for the detailed explanation kind sir